compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/IsolationAnalyzer.java

```diff
@@ -1618,6 +1618,11 @@ public void visit(BLangListConstructorExpr listConstructorExpr) {
         }
     }
 
+    @Override
+    public void visit(BLangListConstructorExpr.BLangListConstructorSpreadOpExpr listConstructorSpreadOpExpr) {
+        analyzeNode(listConstructorSpreadOpExpr.expr, env);
```

Comment:
Can't we get rid of the special in the `BLangListConstructorExpr` visit with this?